doc/flows.puml

```diff
@@ -0,0 +1,222 @@
+' ###############################################################################
+' ########### HERE IS THE LIST OF DIAGRAMS USED FOR THE DOCUMENTATION ###########
+' ###############################################################################
+
+
+' ===========================================================
+' ======================= OAuth2 flow =======================
+' ===========================================================
+
+@startuml
+
+skinparam NoteBackgroundColor #fff
+skinparam NoteBorderColor #000
+
+skinparam sequence {
+	ArrowColor #000
+	ActorBorderColor #000
+
+	LifeLineBorderColor #000
+	LifeLineBackgroundColor #000
+
+	ParticipantBorderColor #000
+	ParticipantBackgroundColor #fff
+	ParticipantFontColor #000
+
+	ActorBackgroundColor #000
+	ActorFontColor #000
+
+        NoteBorderColor #000
+}
+
+skinparam roundcorner 5
+
+actor User                         as user
+participant "Account Web Client"   as account
+database MongoDB                   as mongo
+participant "Connect"              as connect
+
+user -> account : Wants to login
+
+== OAuth2 client initialization ==
+
+account -> connect : Fetch OpenID configuration
+note left
+The **@fwl/oauth2** package mainly provides
+an abstraction for the OAuth2 flow, and a
+secure way to verify JWT for **HS256** and
+**RS256** algorithms
+end note
+connect -> account : Send back OpenID configuration and stores it in-memory
+
+== Connect authentification flow ==
```

Comment:
I think `authentification` is only valid in French 😁 

```suggestion
== Connect authentication flow ==
```

Reply:
Aaahhh nice catch haha 😅.

Reply:
Fixed (and I also fixed the typo in the diagram)
ab81795